armada-ths/ais
# Banquet ticket type model + template update

Added banquet ticket type model. The reason is that we don't want them to be hard coded, so that the banquet peeps can administrate this by themselves. The model itself simply has one field: name. Also id some updates to banquet attendants list template.

Test:
- Go to admin view and add a new banquet ticket type
- Go to /fairs/2017/banquet. To the right of Exhibitor column I have added columns: Ticket type, Table, Seat, Confirmed - make sure they are there! If they have no values they should be empty
- Go to one attendant and change values for new columns and make sure it shows up in the list. 
- You should also be able to choose ticket type from the ticket types you created.

HOPE that was everything :))))
